trin-core/src/jsonrpc/endpoints.rs

```diff
@@ -37,6 +39,7 @@ pub enum InfuraEndpoint {
 #[derive(Debug, PartialEq, Clone)]
 pub enum PortalEndpoint {
     ClientVersion, // Doesn't actually rely on portal network data, but it makes sense to live here
+    GetBlockByHash,
 }
```

Comment:
I came up with `PortalEndpoint` a while back. If I remember correctly, I didn't go with something like `EthEndpoint` because technically, `ClientVersion` is actually `web3_clientVersion` and it seems likely that we might end up supporting other namespaced endpoints from the jsonrpc api (eg. `debug_*`). But I see the confusion now, especially since we've gone with the `portal_*` namespace. I'm on board for `EthEndpoint`, but thought I'd bring this edge case up. Maybe there's a better name? But I can't think of one right now...

Reply:
Yeah, this makes sense to me - but feels out of scope for this pr since it's going to touch a fair amount of the codebase - tracking it here https://github.com/ethereum/trin/issues/273